### ISSUE TITLE
download.topo() error

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1143]
I was trying to get 1-minute resolution around Halifax, but got an error.

```R
> download.topo(-64,-63,44,45,1)
trying URL 'http://maps.ngdc.noaa.gov/mapviewer-support/wcs-proxy/wcs.groovy?filename=etopo1.grd&request=getcoverage&version=1.0.0&service=wcs&coverage=etopo1&CRS=EPSG:4326&format=gmt&resx=0.016667&resy=0.016667&bbox=-64.000000,44.000000,-63.000000,45.000000'
Error in download.file(url, destination) : 
  cannot open URL 'http://maps.ngdc.noaa.gov/mapviewer-support/wcs-proxy/wcs.groovy?filename=etopo1.grd&request=getcoverage&version=1.0.0&service=wcs&coverage=etopo1&CRS=EPSG:4326&format=gmt&resx=0.016667&resy=0.016667&bbox=-64.000000,44.000000,-63.000000,45.000000'
```

**Comment by dankelley** [comment]
Maybe they changed the server, since this worked previously. I see that `marmap` uses a url as below, so I'll try that. 
```R
        WEB.REQUEST <- paste("http://mapserver.ngdc.noaa.gov/cgi-bin/public/wcs/etopo1.xyz?filename=etopo1.xyz&request=getcoverage&version=1.0.0&service=wcs&coverage=etopo1&CRS=EPSG:4326&format=xyz&resx=", 
            res, "&resy=", res, "&bbox=", x1, ",", y1, ",", x2, 
```

**Comment by dankelley** [comment]
Fixed in `develop` commit 4d6bd41d86c264a8d899fdf3600c42d2e736fb57
